internal/profile/hash.go

```diff
@@ -72,6 +72,11 @@ func Hash(p profile.Profile) (string, error) {
 		hasher.Write([]byte("installer tarball fix #10278"))
 	}
 
+	// 8. Images with overlays do not import properly profile overrides.
```

Comment:
Let's also add a comment to issue https://github.com/siderolabs/talos/issues/11093

Reply:
thanks, fixed it up